fairlearn/metrics/_metric_frame.py

```diff
@@ -189,6 +189,41 @@ class MetricFrame:
 
         .. deprecated:: 0.7.0
             `metric` will be removed in version 0.10.0, use `metrics` instead.
+
+    Examples
```

Comment:
I think that it is really important to have good docstring examples for `MetricFrame`. What would you think about essentially just copy-pasting the section "Fairness assessment with MetricFrame" from `https://github.com/fairlearn/talks/blob/main/2021_scipy_tutorial/fairness-in-AI-systems-instructors.ipynb` with a small inline-constructed dataset? This would be longer than our usual docstrings, but not too long by pandas standards.

Reply:
I added the code from the tutorial to the examples. Is this okay?